transmogrifier/models.py

```diff
@@ -35,6 +35,12 @@ def list_of(item_type: Any) -> Callable:  # noqa: ANN401
     )
 
 
+def dedupe(item_list: list | Any) -> list | None:  # noqa: ANN401
```

Comment:
The `item_list: list | Any` type hint is to support scenarios where [a non-list type is set for a list field](https://github.com/MITLibraries/transmogrifier/pull/206/files#diff-0adb667cd397bd56edce10eec11e1b10821740a10d72bd148b09645fc9108968R368-R373). Since converters are applied _before_ validators, the `dedupe` converter is set up to "pass" these values so that the **validator** can effectively handle the values.

Reply:
Smart!